<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="left_group">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/resources/background5.jpg);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>531</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="Dashboard">
        <property name="styleSheet">
         <string notr="true">color: rgb(230, 230, 230);
font: 75 18pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>MyLib</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Dashboard_group_flter">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>1061</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/resources/background4.jpg);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="MyLib_filter_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>Data Filter</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>9</y>
       <width>961</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="MyLib_filter_fileSelector">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/select.png</normaloff>:/new/prefix1/resources/select.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="MyLib_filter_start">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/play-button.png</normaloff>:/new/prefix1/resources/play-button.png</iconset>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTextBrowser" name="MyLib_filter_textview">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QProgressBar" name="MyLib_filter_progressbar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/settings.png</normaloff>:/new/prefix1/resources/settings.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/funnel.png</normaloff>:/new/prefix1/resources/funnel.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>1061</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/resources/background4.jpg);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="MyLib_BookM_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>Book Management</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>561</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_importDB">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>Import database</string>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/import.png</normaloff>:/new/prefix1/resources/import.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_checkOut">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>Check out</string>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/Check_out.png</normaloff>:/new/prefix1/resources/Check_out.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_checkIn">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>Check In</string>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/Check_in.png</normaloff>:/new/prefix1/resources/Check_in.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_search">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>Book Search and Availability</string>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/search.png</normaloff>:/new/prefix1/resources/search.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>1061</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/resources/background4.jpg);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="MyLib_BorrM_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>Borrower Management</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>561</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_createBorr">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>Create Borrower</string>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/create_user.png</normaloff>:/new/prefix1/resources/create_user.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_modifyBorr">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>Modify Borrower</string>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/edit.png</normaloff>:/new/prefix1/resources/edit.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_viewBorr">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>View Borrower</string>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/show.png</normaloff>:/new/prefix1/resources/show.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_deleteBorr">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>Delete Borrower</string>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/user.png</normaloff>:/new/prefix1/resources/user.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>1061</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/resources/background4.jpg);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="MyLib_fineM_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>Fine Management</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>561</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_checkFIne">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>Check Fine</string>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/check_fine.png</normaloff>:/new/prefix1/resources/check_fine.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_setting">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>Setting</string>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/settings.png</normaloff>:/new/prefix1/resources/settings.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_checkOverdue">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Bitstream Charter&quot;;</string>
        </property>
        <property name="text">
         <string>Check Overdue</string>
        </property>
        <property name="icon">
         <iconset resource="rs.qrc">
          <normaloff>:/new/prefix1/resources/deadline-head-symbol.png</normaloff>:/new/prefix1/resources/deadline-head-symbol.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rs.qrc"/>
 </resources>
 <connections/>
</ui>
